gui/src/components/Sidebar.tsx

```diff
@@ -65,24 +67,71 @@ export function SidebarLayout({ children }: { children: ReactNode }) {
   const { theme } = useTheme();
   const breakpoint = theme.breakpoints.down("sm");
 
+  const { settings } = useSettings();
+  const fastMode = settings?.fastMode;
+
+  const { wallets } = useWallets();
+
   const handleKeyboardNavigation = (event: KeyboardEvent) => {
     setLocation(TABS[parseInt(event.key) - 1].path);
   };
 
+  const handleWalletsNavigation = (event: KeyboardEvent) => {
+    invoke("wallets_set_current_wallet", { idx: parseInt(event.key) - 1 });
+  };
+
+  const handleFastModeToggle = () => {
+    invoke("settings_set_fast_mode", { mode: !fastMode });
+  };
+
+  const handleOpenCloseSettings = (event: KeyboardEvent) => {
+    event.preventDefault();
+    const { show } = useSettingsWindow.getState();
+    if (show) {
+      useSettingsWindow.getState().close();
+    } else {
+      useSettingsWindow.getState().open();
+    }
+  };
+
   useMenuAction((payload) => setLocation(payload));
 
+  //------------------------------------------------- Switch between sidebar menus (Ctrl + number)
   useKeyPress(
     range(1, TABS.length + 1).map(toString),
     { meta: true },
     handleKeyboardNavigation,
   );
-
   useKeyPress(
     range(1, TABS.length + 1).map(toString),
     { ctrl: true },
     handleKeyboardNavigation,
   );
 
+  //------------------------------------------------- Switch between wallets (Ctrl + Shift + number)
+  useKeyPress(
+    range(1, Math.min(wallets.length + 1, MAX_CHANGE_WALLET_SHORTCUTS)).map(
```

Comment:
What is the purpose of the call to `map(toString)`?

Reply:
I'm using the `map(toString)` because the custom hook `useKeyPress` expects its first argument to be an array of strings. If I didn't call the `map(toString)`, the `range()` function would yield an array of numbers

Reply:
A good alternative here would be to change the type of the first argument of `useKeyPress` to be an array of string OR an array of numbers, so we don't have to convert to string whenever we use number ranges. Just make sure to convert the keys of the array to string when comparing them to the event key